scripts/build_requirements.py

```diff
@@ -4,11 +4,11 @@
 
 from utils import Downloadable
 
-POOL_URL = "https://deb.debian.org/debian/pool/main"
+POOL_URL = "http://prdownloads.sourceforge.net"
```

Comment:
This is not an acceptable change, _even though_ this is an official upstream release URL for SWIG, because sourceforge is not a trustworthy organization.  We need to continue retrieving SWIG from a trustworthy third party (such as a Linux distribution) that has audited and re-uploaded the tarball.  I prefer Debian for this because I trust their source-auditing practices, but I see that SWIG 4.0 is not yet available there, so I'd accept any other Linux distro or BSD package repo for now.  An HTTPS-based download URL is also strongly preferred (but not strictly required, since we check the tarball's integrity with a cryptographic hash anyway).

Reply:
SWIG 4, while released, is not available anywhere else (including Debian servers).

Reply:
I'm really not going to budge on this one.  _Nowhere_ else?

Reply:
Will add FedoraProject URL. `https://src.fedoraproject.org/lookaside/pkgs/swig/swig-4.0.1.tar.gz/sha512/595ef01cb83adfa960ceed9c325a9429192549e8d1e9aa3ab35a4301512a61d82e2e89a8c7939c2a5a0811254ea1832a443bd387e11459eb2b0bafc563ad1308/swig-4.0.1.tar.gz`